firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -128,7 +128,8 @@ public InstallationResponse createFirebaseInstallation(
             throw new FirebaseException(readErrorResponse(httpsURLConnection));
```

Comment:
referring to the 4 lines above:
```case 500: retryCount++;```

Is it possible for us to treat all server-errors (i.e. 5XX) like this?

Reply:
Done. PTAL.